task_schedule_occ.cfg

```diff
@@ -11,10 +11,10 @@ loud              0                 # Run loudly or quietly (production mode)
 # Data files and directories.  The *_dir vars can have $ENV{} vars which
 # get interpolated.  (Note lack of task name after TST_DATA because this is just for test).
 
-data_dir     $ENV{SKA_DATA}/taco       # Data file directory
-log_dir      $ENV{SKA_DATA}/taco/logs  # Log file directory
-bin_dir      $ENV{SKA_SHARE}/taco      # Bin dir (optional, see task def'n)
-master_log   taco.log             # Composite master log (created in log_dir)
+data_dir     $ENV{SKA_DATA}/acis_taco       # Data file directory
```

Comment:
I don't recall if the data_dir is really used by anything in the task schedule.  @taldcroft ?

Reply:
Hah.  I was first using the oracle of @taldcroft :smile: 

Looks like data_dir is just used as the reference dir for the task_schedule job management files (heartbeat, heart_attack, etc), so this should be fine as-is.

https://github.com/sot/task_schedule/blob/2744c48c19d9084d8bea1c7058bc2e063549de6f/task_schedule.pl#L91